src/hourly-weather.ts

```diff
@@ -320,15 +322,18 @@ export class HourlyWeatherCard extends LitElement {
     const forecastNotAvailable = !forecast || !forecast.length;
 
     if (numSegments < 1) {
+      if (pending) return;
```

Comment:
No need to block this error since it doesn't look at forecast data, just card config.

Reply:
OK, thanks! Removed this.